apps/full-stack-tests/src/IModelUtils.ts

```diff
@@ -240,15 +240,15 @@ export function createSchemaContext(imodel: IModelConnection | IModelDb | ECDb)
       async getSchemaInfo(schemaKey: Readonly<SchemaKey>, matchType: SchemaMatchType, schemaContext: SchemaContext): Promise<SchemaInfo | undefined> {
         const schemaJson = imodel.getSchemaProps(schemaKey.name);
         const schemaInfo = await Schema.startLoadingFromJson(schemaJson, schemaContext);
-        if (schemaInfo !== undefined && schemaInfo.schemaKey.matches(schemaKey, matchType)) {
+        if (schemaInfo !== undefined && schemaInfo.schemaKey.matches(schemaKey as SchemaKey, matchType)) {
```

Comment:
Can we change function definition from `schemaKey: Readonly<SchemaKey>` to `schemaKey: SchemaKey`?